js/src/get-started-page/benefits-card/index.scss

```diff
@@ -0,0 +1,57 @@
+.gla-get-started-benefits-card {
+	// Adjust <CardBody> imported from @wordpress/components.
+	// Repeat selector to make it higher priority.
+	.components-card__body.components-card__body {
+		display: flex;
+		flex-direction: row;
+		padding: 0;
+
+		@media (max-width: $break-small) {
+			flex-direction: column;
+		}
+
+		.components-flex__item {
+			display: flex;
+			flex-direction: column;
+			justify-content: center;
+			padding: $grid-unit-50;
+			max-width: 51.5%;
+
+			> p {
+				margin-bottom: $grid-unit-20;
+				padding: 0 $grid-unit-10;
+
+				&:last-child {
+					margin-bottom: unset;
+				}
```

Comment:
Thanks and TIL for Flexbox's `gap`, updated in 22271b1.

> since the width of this layout is dynamic, so it's okay not to accurately align to the same visual design.

Gotcha, I was desperately trying to make it looks like the design 🤣

Reply:
> I was desperately trying to make it looks like the design

In the case of dynamic content with a dynamic layout, using only CSS could easily run into its limits and not achieve pixel-perfect results. 🙈